web_view_searchpanel/static/src/js/kanban_controller.js

```diff
@@ -0,0 +1,58 @@
+/**
+*
+*    Copyright 2017-2019 MuK IT GmbH.
+*    License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+*
+**/
+
+odoo.define('muk_web_searchpanel.KanbanController', function (require) {
```

Comment:
nitpicking : could you replace ``muk_web_searchpanel`` by ``web_view_searchpanel`` ?